howto: New guide Add Black and isort to an existing Python repository

odscjames opened the pull request:
https://github.com/OpenDataServices/developer-docs/issues/80

jarofgreen committed:
howto: New guide Add Black and isort to an existing Python repository
https://github.com/OpenDataServices/developer-docs/issues/80

odscjames commented:
Do merge if happy

Bjwebb commented on howto/add-black-isort-existing-repo.md:
````diff
@@ -0,0 +1,107 @@
+```
+extras_require_test = ["pytest", "flake8", "isort"]
+
+if sys.version_info[0] >= 3 and sys.version_info[1] >= 6:
````
This will fail for 4.0. Probably better to do `sys.version_info >= (3, 6)`

jarofgreen committed:
howto: New guide Add Black and isort to an existing Python repository
    https://github.com/OpenDataServices/developer-docs/issues/80

Feedback from Ben: https://github.com/OpenDataServices/developer-docs/pull/87

odscjames commented:
```
Python 3.6.9 (default, Nov  7 2019, 10:44:02) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> sys.version_info >= (3, 6)
True
>>> sys.version_info >= (3, 5)
True
>>> sys.version_info >= (3, 7)
False
>>> sys.version_info >= (4, 0)
False
>>> (4,0) >= (3,6)
True
>>> (3,7) >= (3,6)
True
>>> (3,5) >= (3,6)
False

```
